task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -59,6 +59,13 @@ spec:
     description: The image is built from this commit.
     name: COMMIT_SHA
     type: string
+  - default: repos.d
+    description: Path in the repository in which yum repodir files are stored
+    name: REPOSDIR_SRC
+  - default: /etc/yum.repos.d
+    description: Target path on the container in which repodir files should be made
+      available
+    name: REPOSDIR_TARGET
```

Comment:
My preference is, using `YUM_REPOS_D` is much straightforward than `REPOSDIR`.

Then, `REPOSDIR_SRC` -> `YUM_REPOS_D_SRC`, `REPOSDIR_TARGET` -> `YUM_REPOS_D_TARGET`

Reply:
I'll look into that